compose.yaml

```diff
@@ -259,5 +255,5 @@ services:
 volumes:
   openc3-redis-v: {}
   openc3-redis-ephemeral-v: {}
-  openc3-minio-v: {}
+  openc3-bucket-v: {}
```

Comment:
Isn't this a breaking change with a new volume name? How do people upgrade?

Reply:
Yes it is breaking, but makes it the same as Enterprise.  Will make upgrading to enterprise easier.